examples/simple_trixi_controller.c

```diff
@@ -0,0 +1,31 @@
+#include <stdio.h>
+
+#include <trixi.h>
+
+int main ( int argc, char *argv[] ) {
+
+    // Initialize Trixi
+    trixi_initialize();
+
+    // An "elixir" file is required to configure the Trixi simulation
+    const char * elixir = "../../LibTrixi.jl/examples/libelixir_demo.jl";
+
+    // Setup the Trixi simulation
+    // We get a handle to use subsequently
+    int handle = trixi_setup_simulation( elixir );
+
+    // Get time step length
+    printf("Current time step length: %f\n", trixi_calculate_dt(handle));
+
+    // Main loop
+    while ( trixi_is_finished( handle ) == 0 ) {
+
+        trixi_step( handle );
+    }
+
+    // Finalize Trixi
+    trixi_finalize( handle );
```

Comment:
I just realize: If we have something like `trixi_initialize_simulation()` which returns a `handle`, we should also have a matching `trixi_finalize_simulation(handle)`. Then, for complete "tear down", we can again use `trixi_finalize()` without any arguments